sinks/backend/prometheus/prometheus.go

```diff
@@ -65,3 +70,31 @@ func Register() bool {
 
 	return true
 }
+
+func createFeatureConfig() []configFeature {
```

Comment:
looks good, this method should probably be part of the Backend interface, as all back ends will implement it right?